techmatters/flex-plugins
# CHI-2320: Resume case

## Description

* Loading a contact into redux via the 'create' or 'load by task' endpoint will also load a case into redux if it is attached to the contact
* If you close a browser session when adding a new contact with a case open, reopening the contact will open the case so it can be saved / cancelled

### Checklist
- N/A New tests added
- N/A Feature flags added
- N/A Strings are localized

### Related Issues
Fixes CHI-2320

### Verification steps

Create a contact
Add a case
Refresh the browser

You should see the case open on the home page with any saved contacts

Repeat with offline contact

Ensure that save & cancel case functionality work after a browser refresh
